### PR TITLE
Keyword extraction notebook movement

### PR DESCRIPTION
**Pull request by manishkr1754**
Moved Keyword Extraction Detailed Notebook to App folder as per old Pull request requirement

### CODE REVIEW

File: data/dataset-metadata.json [hunk 1]
```diff
@@ -108,35 +108,35 @@
...
-                },
-                {
-                  "name": "Module / Sub-module Difficulty Level",
+                  "name": "Difficulty_Level",
                   "description": "The complexity level of the module or sub-module.",
                   "type": "string"
                 },
                 {
-                  "name": "Keywords / Tags / Skills / Interests / Categories",
+                  "name": "Keywords",
```
Comment (neomatrix369) [inline data/dataset-metadata.json:116]: `Keywords` => `Keywords_Tags_Skills_Interests_Categories`

**Review by neomatrix369 — COMMENTED** [review]
I will review this PR once PR #15 and #16 are reviewed, approved and merged as this one may depend on them

**Review by neomatrix369 — COMMENTED** [review]
Query or concern about field, file name change are mentioned in the other PR - its the same one for this PR as well

**Comment by manishkr1754** [comment]
> Query or concern about field, file name change are mentioned in the other PR - its the same one for this PR as well

Field Name for Keywords changed as per PR request

File: data/dataset-metadata.json (hunk 1 above)
Comment (manishkr1754) [reply data/dataset-metadata.json]: Ok Updated Data info as per request

**Review by neomatrix369 — APPROVED** [review]

**Pull request merged**